classical-components-demo/barcode-scanner/src/main/java/io/scanbot/example/ui/BarcodeScannerActivity.kt

```diff
@@ -17,17 +17,21 @@ import io.scanbot.example.R
 import io.scanbot.example.model.BarcodeResultBundle
 import io.scanbot.example.repository.BarcodeResultRepository
 import io.scanbot.example.repository.BarcodeTypeRepository
+import io.scanbot.sdk.AspectRatio
 import io.scanbot.sdk.ScanbotSDK
 import io.scanbot.sdk.SdkLicenseError
 import io.scanbot.sdk.barcode.BarcodeAutoSnappingController
 import io.scanbot.sdk.barcode.BarcodeDetectorFrameHandler
 import io.scanbot.sdk.barcode.entity.BarcodeScanningResult
 import io.scanbot.sdk.camera.*
+import io.scanbot.sdk.ui.camera.FinderAspectRatio
```

Comment:
please check if the example works after updating to SDK 2.1.0

Reply:
it works